### ISSUE TITLE
Authentication credentials seem to time out...? (low-priority/troubleshooting)

### ISSUE THREAD

**Issue by ktarrio** [parevalo/tstools_online#29]
FYI: this is the message I receive when the tool breaks/'times out' (meaning things are loading/writing/working fine for 30 mins. or so and then things suddenly stop writing) 

![expired_login_issue](https://user-images.githubusercontent.com/44216276/54781191-f9b37500-4bf1-11e9-98ae-9b495d196209.png)

**Comment by parevalo** [comment]
Does this happen every time? Does it happen having the computer on all the time and with a stable internet  connection?

**Comment by ktarrio** [comment]
Yes, this occurs with stable wifi and almost every time (for me at least). Typically I will have the tool open, be writing to it, then go to do something else, leaving the notebook open for 20-40 minutes, and when I return I receive this time out issue
